modules/oxide/src/less/theme/components/view/view.less

```diff
@@ -78,6 +78,7 @@
   .tox-view__pane {
     height: 100%;
     padding: @view-pane-padding;
+    position: relative;
```

Comment:
@tiny-ben-tran does this impact the work you're doing?

Reply:
It doesn't seem to